packages/server/src/lib/email.js

```diff
@@ -533,7 +577,13 @@ function yesterday() {
 }
 
 async function buildAndSendGrantDigestEmails(userId, openDate = yesterday()) {
-    console.log(`Building and sending Grants Digest email for user: ${userId} on ${openDate}`);
+    if (!ENABLED_EMAIL_TYPES.includes(tags.emailTypes.grantDigest)) {
```

Comment:
This is an additional short-circuit check that happens earlier than the one above, added simply because this email tends to be more expensive to build and send as a batch digest to many recipients, so we avoid needlessly building emails.